pkgs/servers/web-apps/vikunja/api.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, buildGoModule, fetchFromGitea, mage, writeShellScriptBin, nixosTests }:
+
+buildGoModule rec {
+  pname = "vikunja-api";
+  version = "0.17.1";
+
+  src = fetchFromGitea {
+    domain = "kolaente.dev";
+    owner = "vikunja";
+    repo = "api";
+    rev = "v${version}";
+    sha256 = "sha256-xqC7MaPe5cClMUTSRE3HLTEH3LH1J1bJSdH+1ZOfGo4=";
+  };
+
+  nativeBuildInputs =
+      let
+        fakeGit = writeShellScriptBin "git" ''
+          if [[ $@ = "describe --tags --always --abbrev=10" ]]; then
+              echo "${version}"
+          else
+              >&2 echo "Unknown command: $@"
+              exit 1
+          fi
+        '';
+      in [ fakeGit mage ];
```

Comment:
Could we patch this easier?

Reply:
It would be possible to always output `version`, but that would be kind of a dirty hack in my opinion. At least I don't know a easier way